elijah/template3a-tarot.js

```diff
@@ -0,0 +1,38 @@
+// tarot card constructor
+var TarotCard = (function(){
+
```

Comment:
Tarot cards are looking good! As an improvement, add your isValid functionality to the rest of your methods so that, if a card is invalid, it is not able to return a name, rank, or suit. Good work!